### PR TITLE
Add pending entries to Changelog

### PR DESCRIPTION
**Pull request by softins**
CHANGELOG: SKIP

**Status of this Pull Request**

Ready for approval

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,5 +1,38 @@
 ### 3.9.1dev <- NOTE: The release version number will be 3.10.0 ###
 
+- Build: Updated bundled Qt6 to version 6.4.0 (#2892).
+  (contributed by @github-actions)
+
+- Build: Mac: Keep the signing-related keychain unlocked (#2927).
```
Comment (ann0see) [inline ChangeLog:6]: I thought we had documented this already...

File: ChangeLog [hunk 2]
```diff
@@ -1,5 +1,38 @@
 ### 3.9.1dev <- NOTE: The release version number will be 3.10.0 ###
 
+- Build: Updated bundled Qt6 to version 6.4.0 (#2892).
```
Comment (ann0see) [inline ChangeLog:3]: I believe only the latest version should be in the changelog

File: ChangeLog [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+- Build: Updated bundled Qt6 to version 6.4.2 (#2991).
+  (contributed by @github-actions)
+
+- Build: Updated bundled Qt6 to version 6.4.3 (#3025).
+  (contributed by @github-actions)
+
+- Build: Updated bundled Qt6 to version 6.5.1 (#3070).
+  (contributed by @github-actions)
+
+- Docs: Added JSON-RPC server connection tips (#3101).
```
Comment (ann0see) [inline ChangeLog:24]: Could you please check if this isn't already mentioned?

File: ChangeLog [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+  (contributed by @mcfnord)
+
+- Added trebmuh to create-translation-issues.sh (#3147).
```
Comment (ann0see) [inline ChangeLog:27]:
```suggestion
- Translation: Added trebmuh to create-translation-issues.sh (#3147).
```

File: ChangeLog (hunk 2 above)
Comment (softins) [reply ChangeLog:3]: OK, will edit. I just used the `tools/changelog-helper.sh` script, and it generated these.

File: ChangeLog (hunk 3 above)
Comment (softins) [reply ChangeLog:24]: This was also added by the `tools/changelog-helper.sh` script. I believe it checks whether the PR numbers are already in the ChangeLog. I'll check manually.

File: ChangeLog (hunk 1 above)
Comment (softins) [reply ChangeLog:6]: Will check. Maybe the PR# was not mentioned.

Comment (softins) [reply ChangeLog:6]: Can't see anything relevant already in the ChangeLog

File: ChangeLog (hunk 3 above)
Comment (softins) [reply ChangeLog:24]: There's no existing mention in the ChangeLog

**Review by pljones — APPROVED** [review]
Feel free to merge once any loose ends are cleaned up.

File: ChangeLog (hunk 1 above)
Comment (ann0see) [reply ChangeLog:3]: I think 
> Build: Updated bundled JACK (Windows-only) to version 1.9.22 (#3031). (contributed by @ann0see)

Is it

Comment (ann0see) [reply ChangeLog:3]: Ok. Removed that and some smaller fixes. Will merge now.

**Review by ann0see — APPROVED** [review]

**Pull request merged**